ecosystem/sep-0008.md

```diff
@@ -248,14 +248,20 @@ Name | Type | Description
+###### Following the Action URL
+If the `action_method` field is provided, the client uses the method specified when making the request to the `action_url`.
 
-- In the case that `action_method` is `GET`, any fields requested may be passed as query parameters in the `action_url`.
+If the `action_fields` field is provided, the client may optionally supply the fields if they already have the information so as to reduce the user's need to re-enter the information. If the requested fields contain sensitive or personal information, the server should use `action_method` `POST` so that sensitive information is not transmitted in a URL.
 
-- In the case that `action_method` is `POST`, any fields requested may be passed as JSON in the request body, using content type `application/json`. The server should respond to the `POST` with a HTTP response with status code `200`, content-type `application/json`. The body should contain either an acknowledgement that the `POST` was sufficient and no further action is required, or the next URL that the user should be taken to if action is still required.
+There are two possible values for `action_method`.
 
-   The next URL in the POST case is necessary to support products that are websites and products that are mobile applications. Products that run entirely in the browser can redirect the user to the next URL and the redirect prevents odd user experiences where a user refreshes a page and is prompted to re-post the data. Products that are mobile applications cannot open a system browser with a POST request. They can make the POST request internally, extract the redirect in the response, and open a system browser with the next URL. Separating the next URL also makes it possible for the server to indicate that the fields provided in the POST request were sufficient and no further user interaction is required.
+- In the case that `action_method` is `GET`, or not defined:
+
+   Fields requested may be passed as query parameters in the `action_url`. The URL should be opened in a browser. Any fields passed should allow the server to proceed without collecting user information, or at the least pre-fill the provided information.
+
+- In the case that `action_method` is `POST`:
+
+   Fields requested may be passed as JSON in the request body, using content type `application/json`. The server should respond to the `POST` with a HTTP response with status code `200`, content-type `application/json`. The body should contain either an acknowledgement that the `POST` was sufficient and no further action is required, or the next URL if action is still required. If a next URL is provided, it should be loaded in a browser for the user to complete any action required.
```

Comment:
I don't see how a client would know how to determine if the request was sufficient or not,

- What is the attribute name of the URL provided in the response? 
- What are the attribute(s) that indicate the request was sufficient?

Reply:
Making changes to answer the above questions is probably best done in a separate PR since this one is just for rephrasing the existing functionality.

Reply:
Just using my best knowledge of [this](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0008.md#action-url) section to answer your questions:

> What is the attribute name of the URL provided in the response?

I think it's named `next_url`.

> What are the attribute(s) that indicate the request was sufficient?

If the response doesn't contain `next_url`, it means no further action is required.

Reply:
Ah, I missed that, never mind then!

Reply:
There is also the `result` response parameter that indicates what type of further action is required or not.